
project vendor/mk/
diff --git a/vendor/mk/config/common_full.mk b/vendor/mk/config/common_full.mk
index 5d12550..9d1e4e3 100644
--- a/vendor/mk/config/common_full.mk
+++ b/vendor/mk/config/common_full.mk
@@ -6,20 +6,10 @@
 
 # Optional MK packages
 PRODUCT_PACKAGES += \
-    Galaxy4 \
-    HoloSpiralWallpaper \
-    LiveWallpapers \
-    LiveWallpapersPicker \
-    MagicSmokeWallpapers \
-    NoiseField \
-    PhaseBeam \
-    VisualizationWallpapers \
-    PhotoTable \
     SoundRecorder
 
 # Extra tools in MK
 PRODUCT_PACKAGES += \
-    vim \
     zip \
     unrar \
     curl
